spec/support/capybara.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.configure do |config|
   config.prepend_before(:example, type: :system) do |example|
-    if ENV['CYPERFUL']
+    # If you can't use Cyperful, a headed test can be helpful for authoring system specs.
+    if ENV['CYPERFUL'] || example.metadata[:headed_test]
```

Comment:
Worth tossing something in the README for this?

Reply:
Yes, done.